.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What does this file do?

Reply:
I'm sorry, I forgot to add the folder to gitignore, I will remove it